python/ray/exceptions.py

```diff
@@ -910,6 +907,27 @@ class RayCgraphCapacityExceeded(RaySystemError):
     pass
 
 
+@PublicAPI(stability="alpha")
+class UnserializableException(RayError):
+    """Raised when there is an error deserializing the serialized exception.
+
+    This occurs when deserializing (unpickling) a previously serialized exception
+    fails. In this case, we fall back to raising the string representation of
+    the original exception along with its stack trace that was captured at the
+    time of serialization.
+    """
+
+    def __init__(self, original_stack_trace: str):
+        self.original_stack_trace = original_stack_trace
```

Comment:
```suggestion
        self._original_stack_trace = original_stack_trace
```
private attributes should be underscore prefixed